docker-compose.override.yml

```
@@ -2,7 +2,7 @@ version: "3.4"
 
 services:
 
-    python2:
+    python3:
```

Should we rename this to just "python"? Before we had two Python containers, the Python 2 container was just called "python".

```suggestion
    python:
```

---

If the answer to this is yes, then lots of other changes that I didn't explicitly suggest below would naturally flow from that decision.